aten/src/ATen/DLConvertor.cpp

```diff
@@ -306,7 +306,7 @@ Tensor fromDLPack(const DLManagedTensor* src) {
 
 Tensor fromDLPack(
     const DLManagedTensor* src,
-    std::function<void(void*)> deleter) {
+    const std::function<void(void*)>& deleter) {
```

Comment:
from_blog can't take moved functions?

Reply:
from_blog takes deleter as const &, I will change it in the next PR.